testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java

```diff
@@ -202,6 +202,7 @@ public void diacriticsJVM(TestInfo testInfo) throws IOException, InterruptedExce
     }
 
     @Test
+    @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/9707
```

Comment:
good point, I think to make this work you need to have Czech language support installed on Windows, so by default this test doesn't work.